app/models/address.rb

```diff
@@ -7,13 +7,30 @@ class Address < Struct.new(
   :county,
   :postcode
 )
+  def blank?
+    attributes.none?(&:present?)
+  end
+
   def to_s
+    attributes.compact_blank.join(", ")
+  end
+
+  def as_json(*)
+    to_s
+  end
+
+  alias_method :to_json, :as_json
+  alias_method :to_str, :to_s
```

Comment:
The `to_str` is so that Ruby will coerce the value into a string - otherwise code like this might fail:

``` ruby
"(" + planning_application.address + ")"
```

The json methods are so that the API endpoints encode the address as a string and not an object.